packages/react/src/contents/hooks/useSeoFiles.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  fetchSEOFiles,
+  getSEOFilesError,
+  getSEOFilesResult,
+  isSEOFilesFetched,
+  isSEOFilesLoading,
+  type StoreState,
+} from '@farfetch/blackout-redux';
+import { useCallback, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import useAction from '../../helpers/useAction.js';
+import type { UseSeoFilesOptions } from './types/useSeoFiles.types.js';
+
+/**
+ * Hook to return seo metadata to populate document head.
+ *
+ * @param data - object that receives SEO query, icon links, additional links, metas and options
+ *
+ * @returns - Returns actions and SEO metadata state for a given path.
+ */
+const useSeoFiles = ({ query, enableAutoFetch = true }: UseSeoFilesOptions) => {
```

Comment:
Instead of using only one parameter, spread this into 2 like this:
```suggestion
const useSeoFiles = (query: GetSEOFilesQuery, options: UseSeoFilesOptions = { enableAutoFetch = true }) => {
```

Reply:
Resolved! ✅